.changeset/pre.json

```diff
@@ -0,0 +1,8 @@
+{
+  "mode": "pre",
+  "tag": "rc",
```

Comment:
This will also publish the npm package under the `rc` dist-tag.  If we wanted to have a `2025-10-rc` tag, we could look into adjusting the ` Set 'latest' NPM dist tag` step to also deal with rc versions, but I'd like to see the whole thing work first before investing into this.